setup.cfg

```
@@ -84,7 +84,8 @@ parquet =
 feather =
     pyarrow>=6.0.0
 hdf5 =
-    blosc>=1.20.1
+    # blosc only available on conda
```

To save future readers' time it would be helpful to point to the evidence/discussion on this point
```suggestion
    # blosc only available on conda (https://github.com/Blosc/python-blosc/issues/297)
```